app/mailers/generic_mailer.rb

```diff
@@ -140,11 +140,274 @@ def self.default_for_enable_starttls_auto
   end
 
   def self.openssl_verify_modes
-    %w(none peer client_once fail_if_no_peer_cert)
+    [
+      [_("None"),                 "none"],
+      [_("Peer"),                 "peer"],
+      [_("Client Once"),          "client_once"],
+      [_("Fail If No Peer Cert"), "fail_if_no_peer_cert"]
+    ]
   end
 
   def self.authentication_modes
-    %w(login plain none)
+    [[_("login"), "login"], [_("plrequire: rubocop-performance
+
+AllCops:
```

Comment:
I think this was accidentally pasted here.

Reply:
Ha...  Yeah.